sys-boot/efibootmgr/efibootmgr-17.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="User-space application to modify the EFI boot manager"
+HOMEPAGE="https://github.com/rhinstaller/efibootmgr"
+SRC_URI="https://github.com/rhboot/efibootmgr/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~x86"
+
+RDEPEND="
+	sys-apps/pciutils
+	>=sys-libs/efivar-37:=
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-clang.patch" )
+
+src_prepare() {
+	default
+	sed -i 's/-Werror //' Make.defaults || die
+	# Remove conflict with =sys-libs/efivar-37.
+	# See: https://github.com/rhboot/efibootmgr/issues/128
+	sed -i '/extern int efi_set_verbose/d' src/efibootmgr.c || die
+}
+
+src_configure() {
+	tc-export CC
+	export EFIDIR="Gentoo"
+}
+
+src_compile() {
+	PKG_CONFIG="$(tc-getPKG_CONFIG)" emake
```

Comment:
It looks like you swapped the order of emake and PKG_CONFIG here. This has a negative effect on the build process.

In Make.defaults, PKG_CONFIG is set using the `=` operator. This means that environment variables will not override it, but command line arguments will.

Reply:
Sorry, I meant to put that back.